governance/votes/Executive Vote - June 27, 2024.md

```diff
@@ -0,0 +1,124 @@
+  
+- **Authorization:** [Accessibility Scope 9.1](https://mips.makerdao.com/mips/details/MIP108#9-1-launch-project-budget)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/18)
+  
+If this executive proposal passes, the following transfers to the Launch Project will be made:
+  
+- Transfer **4,500,000 DAI** to the Launch Project at [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
+- Transfer **1,300 MKR** to the Launch Project at [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
+  
+### Spark Proxy Spell
+  
+- **Authorization:** Governance Polls [1119](https://vote.makerdao.com/polling/QmQv9zQR), [1117](https://vote.makerdao.com/polling/QmdQYTQe), [1118](https://vote.makerdao.com/polling/QmU6KSGc)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/jun-12-2024-proposed-changes-to-sparklend-for-upcoming-spell/24489)
+  
+If this executive proposal passes, a Spark proxy spell at [0xc96420Dbe9568e2a65DD57daAD069FDEd37265fa](https://etherscan.io/address/0xc96420Dbe9568e2a65DD57daAD069FDEd37265fa) will be triggered, containing the following items.
+  
+#### [Mainnet] Update Morpho Vault Supply Caps
+  
+The following changes will be made to the [Morpho Spark DAI Vault](https://morpho.blockanalitica.com/ethereum/metamorpho/vaults/0x73e65dbd630f90604062f6e02fab9138e713edd9) parameters:
+  
+- [sUSDe 91.5% LLTV pool](https://morpho.blockanalitica.com/ethereum/markets/1247f1c237eceae0602eab1470a5061a6dd8f734ba88c7cdc5d6109fb0026b28) supply cap will be increased by 100 million DAI from 100 million DAI to **200 million DAI**.
+- [sUSDe 86% LLTV pool](https://morpho.blockanalitica.com/ethereum/markets/39d11026eae1c6ec02aa4c0910778664089cdd97c3fd23f68f7cd05e2e95af48) supply cap will be increased by 100 million DAI from 400 million DAI to **500 million DAI**.
+
+#### [Gnosis] Onboard USDC.e as Borrowable Asset
+
+Bridged USDC (USDC.e) will be onboarded with the following parameters:
+
+- Token: **USDC.e**
+- Token Address: **[0x2a22f9c3b484c3629090FeED35F17Ff8F88f76F0](https://gnosisscan.io/token/0x2a22f9c3b484c3629090feed35f17ff8f88f76f0)**
+- Oracle: **1:1 USD Fixed**
+- Oracle Provider: N/A
+- Optimal Utilization: **95%**
+- Base: **0%**
+- Slope1: **9%**
+- Slope2: **15%**
+- Stable rate func (deprecated): N/A
+- Borrow Enabled: **Yes**
+- Isolation Borrowable: **Yes**
+- Stable Rate Borrowing: **No**
+- [Siloed Borrowing](https://docs.spark.fi/defi-infrastructure/sparklend#siloed-borrowing): **No**
+- Flash Loan Enabled: **Yes**
+- Loan To Value: **0%**
+- Liquidation Threshold: **0%**
+- Liquidation Bonus: **0%**
+- Reserve Factor: **10%**
+- Cap Automator: **Disabled**
+- [Supply Cap](https://docs.spark.fi/defi-infrastructure/sparklend#supply-and-borrow-caps): **10,000,000 USDC.e**
+- Supply Cap (`max`): N/A
```

Comment:
These `max`, `gap`, and `ttl` parameters were left off of the poll intentionally because they do not provide additional information. Additionally, the way BA presented the information would be confusing to a regular voter. We utilise the following parameter notation -- $parameter_human_friendly_name ($name_in_code) 

BA Labs have shortened our naming convention in the post, it is not the "Supply Cap (`max`), rather  "Supply Cap Maximum Debt Ceiling (`max`)" and so on for each parameter listed. I recommend taking this information out completely, but if you wish to leave it in the names should be corrected to prevent voter confusion.